Cargo.toml

```diff
@@ -17,3 +17,4 @@ include = ["src/**/*", "README.md"]
 lazy_static = { version = "1.4.0" }
 maplit = { version = "1.0.2" }
 strum = { version = "0.24.0", features = ["derive"] }
+terminal-log-symbols = "0.1.6"
```

Comment:
I would prefer not to add another dependency, could we just keep your changes on stop_and_persist and stop_with_message? Thanks :)

Reply:
Ok, I will revert the commits

Reply:
Done